README.md

```diff
@@ -62,12 +62,13 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
    }
    ```
 2. Add a [custom option](https://cloud.google.com/service-management/reference/rpc/google.api#http) to the .proto file
-   
+   Also you can customize every http handler with middleware. Just add gengo.grpc.gateway.middleware option.
```

Comment:
We should document this in a distinct section and keep the original example simple.